frappe/public/js/frappe/ui/dialog.js

```diff
@@ -232,6 +232,8 @@ frappe.ui.Dialog = class Dialog extends frappe.ui.FieldGroup {
 		this.get_minimize_btn().html(frappe.utils.icon(icon));
 		this.on_minimize_toggle && this.on_minimize_toggle(this.is_minimized);
 		this.header.find('.modal-title').toggleClass('cursor-pointer');
+		var body = $("body");
+		body.hasClass("modal-open") ? body.removeClass("modal-open") : body.addClass("modal-open");
```

Comment:
```suggestion
		$("body").toggleClass("modal-open", !this.is_minimized);
```

Reply:
Also, doing this added a horizontal scrollbar on my local site.

Reply:
The horizontal scroll bar was appearing because on removing modal-open class [this](https://github.com/frappe/frappe/blob/e22288ec8d0f8722ed926e8cd125a2c2190ab67c/frappe/public/scss/common/modal.scss#L5) padding-right was getting removed as well.

So now I feel that a better solution would be to not toggle the modal-open class and to just change the overflow values.